POSTGAP.pm

```diff
@@ -126,20 +126,22 @@ sub new {
 
   # get required columns
   my $all =0;
-  my $i = 4;
+  my $i = 1;
   while(defined($self->params->[$i])) {
     my $col = $self->params->[$i];
     if($col eq 'ALL') {
+      $i = 4;
```

Comment:
Sorry, I am not sure I understand this line, knowing that you use the `3 + $i` later in the code.
I know you said that you want the 3 first columns to be "fixed" with the same attributes.
Would it be possible to add some comments/examples ?